src/graph.js

```diff
@@ -60,6 +61,19 @@ module.exports = (function () {
         return this;
     };
 
+    Graph.prototype.dontRender = function() {
+        this._renderNow = false;
+
+        return this;
+    };
+
+    Graph.prototype.doRender = function() {
```

Comment:
Another idea: how about instead of these two methods, we have a new **public** property ([here](https://github.com/frewsxcv/graphosaurus/blob/master/src/graph.js#L33-L61) is where the other properties get set; the new one doesn't need a `_` prefix), so the user can just update/set that to specify whether they want to rerender upon mutation? We could call it something like `renderOnChange` or `autoRerender`.